packages/spor-react/src/theme/recipes/skeleton.ts

```diff
@@ -0,0 +1,52 @@
+import { defineRecipe } from "@chakra-ui/react";
+
+export const skeletonRecipe = defineRecipe({
+  className: "chakra-skeleton",
+  variants: {
+    loading: {
+      true: {
+        borderRadius: "xs", 
+        boxShadow: "none",
+        backgroundClip: "padding-box",
+        cursor: "default",
+        color: "transparent",
+        pointerEvents: "none",
+        userSelect: "none",
+        flexShrink: "0",
+        "&::before, &::after, *": {
+          visibility: "hidden",
+        },
+      },
+      false: {
+        background: "unset",
+        animation: "fade-in var(--fade-duration, 0.1s) ease-out !important",
+      },
+    },
+    variant: {
+      pulse: {
+        background: "steel",
```

Comment:
Vi må huske darkmode på denne, foreslår:
```suggestion
        background: {
            _light: "silver",
            _dark: "darkGrey",
        },
```